CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+# Contributing guidelines
+
+- [Introduction](#introduction)
+- [Pull Requests (PRs)](#pull-requests-prs)
+  - [Etiquette for creating PRs](#etiquette-for-creating-prs)
+  - [Checklist for publishing PRs](#checklist-for-publishing-prs)
+- [Setting up a local development
+  environment](#setting-up-a-local-development-environment)
+  - [Code checks with pre-commit](#code-checks-with-pre-commit)
+  - [Unit tests with pytest](#unit-tests-with-pytest)
+
+## Introduction
+
+Thank you for contributing to SHAP. SHAP is an open source collective effort,
+and contributions of all forms are welcome!
+
+You can contribute by:
+
+- Submitting bug reports and features requests on the GitHub issue tracker
+- Contributing fixes and improvements via Pull Requests
+
+## Pull Requests (PRs)
+
+### Etiquette for creating PRs
+
+- Before starting on a PR, please make a proposal by opening an Issue, and check
+  for any duplicates
+- Work on a feature branch with a descriptive name
+- Open a Draft PR as early as possible, do not wait until the feature is ready
+
+### Checklist for publishing PRs
+
+Before marking your PR as "ready for review" (by removing the `Draft` status),
+please ensure:
+
+- All [pre-commit hooks](https://pre-commit.com/#install) pass
+- If your PR adds any new features or fixes a bug, ensure unit tests have been
+  added
+- Your feature branch is up-to-date with the master branch
+
+## Setting up a local development environment
+
+1. Fork the repository on Github to your user area
```

Comment:
I'll be a bit more nitpicky in our documentation. Let's make sure to end our sentences with periods so it's a little more formal.